The job **`aarch64-apple`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/26525880929/job/78129379820) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/78129379820) [(plain)](https://github.com/rust-lang/rust/commit/77f48760b670d93e39bd3707364d703707ce7638/checks/78129379820/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
To only update this specific test, also pass `--test-args darwin-objc/darwin-objc-bad-arg.rs`

error: 1 errors occurred comparing output.
status: exit status: 1
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/bin/rustc" "/Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/Users/runner/.cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/Users/runner/work/rust/rust/vendor" "--sysroot" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2" "--target=aarch64-apple-darwin" "--check-cfg" "cfg(test,FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "--emit" "metadata" "-C" "prefer-dynamic" "--out-dir" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/test/ui/darwin-objc/darwin-objc-bad-arg" "-A" "unused" "-W" "unused_attributes" "-A" "internal_features" "-A" "incomplete_features" "-A" "unused_parens" "-A" "unused_braces" "-Crpath" "-Cdebuginfo=0" "-Lnative=/Users/runner/work/rust/rust/build/aarch64-apple-darwin/native/rust-test-helpers" "--edition=2024"
stdout: none
--- stderr -------------------------------
error: attribute value must be a literal
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:12:18
   |
LL |     objc::class!(s);
   |                  ^ help: try adding quotation marks: `"s"`

error: attribute value must be a literal
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:15:18
   |
LL |     objc::class!(NSObject);
   |                  ^^^^^^^^ help: try adding quotation marks: `"NSObject"`

error: `objc::class!` expected a string literal
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:18:18
   |
LL |     objc::class!(123);
   |                  ^^^

error: `objc::class!` may not contain null characters
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:21:18
   |
LL |     objc::class!("NSObject\0");
   |                  ^^^^^^^^^^^^

error: attribute value must be a literal
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:25:21
   |
LL |     objc::selector!(s);
   |                     ^ help: try adding quotation marks: `"s"`

error: attribute value must be a literal
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:28:21
   |
LL |     objc::selector!(alloc);
   |                     ^^^^^ help: try adding quotation marks: `"alloc"`

error: `objc::selector!` expected a string literal
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:31:21
   |
LL |     objc::selector!(123);
   |                     ^^^

error: `objc::selector!` may not contain null characters
##[error]  --> /Users/runner/work/rust/rust/tests/ui/darwin-objc/darwin-objc-bad-arg.rs:34:21
   |
LL |     objc::selector!("alloc\0");
   |                     ^^^^^^^^^

error: aborting due to 8 previous errors
------------------------------------------

```

</details>

--- a/tests/ui/darwin-objc/darwin-objc-bad-arg.stderr
+++ b/tests/ui/darwin-objc/darwin-objc-bad-arg.stderr
@@ -2,13 +2,13 @@ error: attribute value must be a literal
   --> $DIR/darwin-objc-bad-arg.rs:12:18
    |
 LL |     objc::class!(s);
-   |                  ^
+   |                  ^ help: try adding quotation marks: `"s"`
 
 error: attribute value must be a literal
   --> $DIR/darwin-objc-bad-arg.rs:15:18
    |
 LL |     objc::class!(NSObject);
-   |                  ^^^^^^^^
+   |                  ^^^^^^^^ help: try adding quotation marks: `"NSObject"`
 
 error: `objc::class!` expected a string literal
   --> $DIR/darwin-objc-bad-arg.rs:18:18
@@ -26,13 +26,13 @@ error: attribute value must be a literal
   --> $DIR/darwin-objc-bad-arg.rs:25:21
    |
 LL |     objc::selector!(s);
-   |                     ^
+   |                     ^ help: try adding quotation marks: `"s"`
 
 error: attribute value must be a literal
   --> $DIR/darwin-objc-bad-arg.rs:28:21
    |
 LL |     objc::selector!(alloc);
-   |                     ^^^^^
+   |                     ^^^^^ help: try adding quotation marks: `"alloc"`
 
 error: `objc::selector!` expected a string literal
   --> $DIR/darwin-objc-bad-arg.rs:31:21

